system/autoware_diagnostic_graph_aggregator/doc/format/unit/diag.md

```diff
@@ -4,7 +4,11 @@ The `diag` object is a unit that refers to a specific status within the source d
 
 ## Format
 
-| Name   | Type     | Required | Description                            |
-| ------ | -------- | -------- | -------------------------------------- |
-| `type` | `string` | yes      | Specify `diag` when using this object. |
-| `diag` | `string` | yes      | The name of the diagnostic status.     |
+| Name      | Type     | Required | Description                            |
+| --------- | -------- | -------- | -------------------------------------- |
+| `type`    | `string` | yes      | Specify `diag` when using this object. |
+| `node`    | `string` | yes      | The name of the diagnostic status.     |
+| `name`    | `string` | yes      | The name of the diagnostic status.     |
+| `timeout` | `float`  | yes      | The name of the diagnostic status.     |
+
+In normal use cases, `node` parameter should be the `Hardware ID` of a node and the `name` should be one of the diagnosis attributes of that node.
```

Comment:
The `node` parameter should `[node name]`.  Typically the full diagnostics name is `[node-name]: [diag-name]` and these parameters will be used as `<node>: <name>`. The Hardware ID is not used for processing.

Reply:
@isamu-takagi I fixed it, can you review it ?